crates/forge-runner/src/test_case_summary.rs

```diff
@@ -3,13 +3,44 @@ use crate::expected_result::{ExpectedPanicValue, ExpectedTestResult};
 use cairo_felt::Felt252;
 use cairo_lang_runner::short_string::as_cairo_short_string;
 use cairo_lang_runner::{RunResult, RunResultValue};
+use num_integer::Roots;
 use std::option::Option;
 
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Clone, Default)]
 pub struct GasStatistics {
     pub min: u128,
     pub max: u128,
+    pub mean: u128,
+    pub std_deviation: u128,
```

Comment:
Why the fields if this value is calculated? Could we refactor that and use gas_usages only instead?

Reply:
I think it is better to have the results here, we don't need `gas_usages` later on

Reply:
I think it is better to explicitly define what information runner returns, thus keep those fields here. I don't see why this is a problem 🤔 . It is not a concern of the cli to calculate the statistics imo.